twitter_ads/account.py

```diff
@@ -136,6 +136,12 @@ def media_creatives(self, id=None, **kwargs):
         """
         return self._load_resource(MediaCreative, id, **kwargs)
 
+    def video_website_cards(self, id=None, **kwargs):
```

Comment:
I'm not sure if this is needed because the other card formats also use just the RESOURCE_COLLECTION?

Reply:
My understanding is that including this under the `Account` class let's us retrieve the video website cards that belong to the particular account.

For example:

```
In [3]: client = Client(CONSUMER_KEY, CONSUMER_SECRET, ACCESS_TOKEN, ACCESS_TOKEN_SECRET)
   ...: account = client.accounts(ACCOUNT_ID)
   ...: 

In [4]: vwcs = account.video_website_cards()

In [5]: type(vwcs)
Out[5]: twitter_ads.cursor.Cursor

In [6]: vwcs.first.id
Out[6]: u'455hx'

In [7]: vwcs.first.card_uri
Out[7]: u'card://860983079292198912'

In [8]: vwcs.first.card_type
Out[8]: u'VIDEO_WEBSITE'
```

You can do this type of thing for any method (entity) defined under the `Account` class.